### PR TITLE
use map for icons

### PR DESCRIPTION
**Pull request by Monirzadeh**
it is related to #284 
later i will review that code to make that better but for now try to fix golangci-lint error that remain in code 

Note: why i don't use single map? i am not complety read the code right now but if should search in maps multiple time maybe smaller map be better instead of load the full map each time . beside ram usage it can affect search time in map (i will review that later and try to find a better solution for that)

### CODE REVIEW

**Comment by jovandeginste** [comment]
> @jovandeginste do you think we need `gofumpt`? relay `gofmt` is not enough?

I'd rather have `gofumpt` enabled, yes

**Comment by jovandeginste** [comment]
Can we merge all maps into one now, with a single lookup function? I split them originally because the linter complained... If the linter is happy with a single map, we should then sort the map alphabetically...

File: pkg/templatehelpers/icons.go [hunk 1]
```diff
@@ -4,185 +4,90 @@ import "html/template"
...
+	"skiing":         "icon-solid icon-person-skiing",
+	"snowboarding":   "icon-solid icon-person-snowboarding",
+	"golfing":        "icon-solid icon-golf-ball-tee",
+	"kayaking":       "icon-solid icon-sailboat",
+	"hiking":         "icon-solid icon-person-hiking",
+	"push-ups":       "icon-solid icon-dumbbell",
+	"weight lifting": "icon-solid icon-dumbbell",
 
-func sportIcon(what string) string {
-	// We need every icon fully qualified here. Otherwise Tailwind will not pick
```
Comment (jovandeginste) [inline pkg/templatehelpers/icons.go:94]: Can you keep this comment, eg. above the iconMap declaration?

**Review by jovandeginste — CHANGES_REQUESTED** [review]
one final request, then it's good for me

File: pkg/templatehelpers/icons.go (hunk 1 above)
Comment (Monirzadeh) [reply pkg/templatehelpers/icons.go:94]: done

**Review by jovandeginste — APPROVED** [review]

**Comment by jovandeginste** [comment]
Thanks, merged!

**Pull request merged**